docs/usage/projects.md

```diff
@@ -0,0 +1,90 @@
+# Projects
+
+The Gardener API server supports a cluster-scoped `Project` resource which is used to group usage of Gardener.
+For example, each development team has its own project to manage its own shoot clusters.
+
+Each `Project` is backed by a Kubernetes `Namespace` that contains the actual related Kubernetes resources like `Secret`s or `Shoot`s.
+
+**Example resource:**
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Project
+metadata:
+  name: dev
+spec:
+  # The `spec.namespace` field is optional and will be initialized if unset - the resulting
+  # namespace will be generated and look like "garden-dev-<random-chars>", e.g. "garden-dev-5z43z".
+  # If the namespace is set then the namespace must be labelled with `gardener.cloud/role: project`
+  # and `project.gardener.cloud/name: <project-name>` (<project-name>=dev in this case).
+  namespace: garden-dev
+  description: "This is my first project"
+  purpose: "Experimenting with Gardener"
+  owner:
+    apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: john.doe@example.com
+  members:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: alice.doe@example.com
+  # roles: # Additional roles go here
+  # - viewer
+  # - uam # User access manager
+  - apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: bob.doe@example.com
+    role: viewer
+# tolerations:
+#   defaults:
+#   - key: <some-key>
+#   whitelist:
+#   - key: <some-key>
+```
+
+The `.spec.namespace` field is optional and will be initialized if it's unset.
+The name of the resulting namespace will be generated and look like `garden-dev-5anj3`, i.e., it has a random suffix.
+It's also possible to adopt existing namespaces by labeling them `gardener.cloud/role=project` and `project.gardener.cloud/name=dev` beforehand (otherwise, they cannot be adopted). 
+
+The `spec.description` and `.spec.purpose` fields can be used to describe to fellow team members and Gardener operators what this project is used for.
+
+Each project has one dedicated owner, configured via a RBAC subject in `.spec.owner`.
```

Comment:
>  configured via a RBAC subject in `.spec.owner`.

I think this is not necessary.

Reply:
Or at least it is not very clear to me why it is mentioned.

Reply:
Hm, why? I'm explaining the various fields in the `spec`.

Reply:
The first time I was reading the sentence I thought that the owner is configured by `(Cluster)RoleBinding` and should be done outside of the `Project` resource.
A bit later I find out that you want to say that `RBAC subject` is actually the type of the field, e.g. the golang type.

Now, I know It is correct, but for detailed explanation of the field the [API reference](https://gardener.cloud/api-reference/core/#core.gardener.cloud/v1beta1.Project) can be used.

Reply:
OK, I have rephrased it. PTAL :)